package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "sqlcommenter-sequelize",
+  "version": "1.0.0",
+  "description": "Middleware and wrappers to correlate SQL statements from Sequelize.js",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "devDependencies": {
+    "chai": "^4.2.0",
+    "chai-http": "^4.3.0",
+    "chai-jest-snapshot": "^2.0.0",
+    "mocha": "^10.2.0"
+  },
```

Comment:
should i add a dependency here on sequelize? the wrapper only works for the current format of our Sequelize and how it works

Reply:
I don't think we need to, but I do have another idea for a better end-to-end test I'll put in the main MR!